Gopkg.toml

```diff
@@ -96,6 +96,10 @@ ignored = [
   name = "github.com/aws/aws-sdk-go"
   version = "~1.12.48"
 
+[[constraint]]
```

Comment:
Remove this constraint and all `vendor` files - not needed anymore.

Reply:
Done.